flow/environments/umn.py

```diff
@@ -0,0 +1,15 @@
+"""Environment for the Minnesota Supercomputing Institute at University of Minnesota"""
```

Comment:
Missing copyright header -- please copy this from another file and update the date to 2020.